Directory.Build.props

```diff
@@ -38,7 +38,7 @@
     <GeneratePackageOnBuild>false</GeneratePackageOnBuild>
     <!-- The release version should get updated on every new release -->
     <Version>20.0.3</Version>
-    <PreReleaseSuffix Condition="'$(PreReleaseSuffix)' == ''">local</PreReleaseSuffix>
+    <PreReleaseSuffix Condition="'$(PreReleaseSuffix)' == ''">$([System.DateTime]::Now.ToString(yyyyMMdd-HHmmss))</PreReleaseSuffix>
```

Comment:
`UtcNow` would be better in my view that `Now` as it is consistent irrespective of the timezone in use. It's also about 30 times more efficient

Reply:
Good point, updated, thank you